### ISSUE TITLE
Keyboard navigation on the igx-grid doesn't work when igx-grid is in the ig-tabs

### ISSUE THREAD

**Issue by tkiryu** [IgniteUI/igniteui-angular#1106]
### Description  
Keyboard navigation on the igx-grid doesn't work when igx-grid is in the ig-tabs.

### Steps to reproduce  

1. Go to https://stackblitz.com/edit/igx-grid-in-igx-tabs
2. Click a cell.
3. Press arrow left/right key.

### Result  
Tabs switches.

### Expected result  
Keyboard navigation on the igx-grid should be priority.

**Comment by SlavUI** [comment]
@tkiryu The sample you provided can't be open :)

**Comment by MayaKirova** [comment]
I looked into this and it seems that the igx-tabs changes the active tab on arrow left/right when something in the content panel is focused regardless of whether it’s a grid or something else.
For instance, here’s a sample with tabs with simple input element:
https://stackblitz.com/edit/igx-grid-in-igx-tabs-b1j5yx?file=app/app.component.html
Note that once the focus is on the input arrow left/right will change the active tab, disallowing you to change the cursor position inside the input.

It should change the active tab only when the actual tab header element is focused. Actions done on the content  should not affect it.

As this does not seem to be related to the grid I am removing the grid tag and assigning it to the related team.

**Comment by sboykova** [comment]
Fixed and merged
